SPECS-EXTENDED/ganglia/ganglia.signatures.json

```diff
@@ -0,0 +1,11 @@
+{
+ "Signatures": {
+  "ganglia-web-3.7.5.tar.gz": "067f791287a11210d274acf4c9c1b5e3df3482ab3f3a151071cf6c926460cfa9",
```

Comment:
I do not think the sources are uploaded to the blobstore.
I can upload them on your behalf.

Reply:
yes..Thanks